src/aws/osml/features/imaged_feature_property_accessor.py

```diff
@@ -0,0 +1,136 @@
+import json
+from typing import Optional
+
+import geojson
+import shapely
+
+
+class ImagedFeaturePropertyAccessor:
+    """
+    This class contains utility functions that ensure the property names / values for features derived from imagery
+    are consistently implemented. These specifications are still evolving so the intent is to encapsulate all of the
+    names in this one class so that changes do not ripple through the rest of the software baseline.
+    """
+
+    IMAGE_GEOMETRY = "imageGeometry"
+    IMAGE_BBOX = "imageBBox"
+
+    BOUNDS_IMCORDS = "bounds_imcoords"
+    GEOM_IMCOORDS = "geom_imcoords"
+    DETECTION = "detection"
+    TYPE = "type"
+    COORDINATES = "coordinates"
+    PIXEL_COORDINATES = "pixelCoordinates"
```

Comment:
Looks like we have a mix of camelCase and snake_case values - would we be able to standardize on one of them or is this driven by an external interface?

Reply:
These are driven by external interfaces. 

I agree it would be best to pick one convention and stick with it. At some point we may want to release a breaking change to remove the older values.